src/components/pages/Gallery/GalleryContainer.js

```diff
@@ -13,7 +13,7 @@ import { useHistory, useParams } from 'react-router-dom';
 const GalleryContainer = props => {
   const { user } = useAuth0();
   const { push } = useHistory();
-  const [data, setDataInfo] = useState([]);
+  const [, setDataInfo] = useState([]);
```

Comment:
Same feedback about getting rid of state data, I assume I don't know this technique and this works fine. 
But just wanted to make sure with you

Reply:
Yep